### DISCUSSION TITLE
Onlyoffice slow loading : related to caddy that gives 502 on wss://cloud.domain.tld/push/ws ?

### DISCUSSION THREAD

**Discussion by t3cneo** [nextcloud/all-in-one#3858] [Questions] [answered]
Hi guys

I updated to latest yesterday and noticed onlyoffice takes ages to load any document in a browser (Firefox).

Trying to load/create a document finally works right after some kind of timout on `wss://cloud.domain.tld/onlyoffice/7.5.1-23/doc/12345678/c/?EIO=4&transport=websocket`

Here is what I have in the dev tools :

![Capture d’écran 2023-12-02 à 15 25 41](https://github.com/nextcloud/all-in-one/assets/42748615/f45cbdc9-85c0-43ad-ada8-a9b7531be8e5)

Appart from that I didn't notice any other side effects so far, notifications seems to be working absolutely fine.

Any of you experiencing the same wss:// 502 error or is it just me?

Could it explain why onlyoffice is slow to load a document?

**Comment by szaimen** [comment, marked as answer]
Hi, do you run AIO behind a reverse proxy?

**Reply by t3cneo** [reply]
Hello

Nope, I run the manual install with the docker-compose (and .env) file [found on the repo](https://github.com/nextcloud/all-in-one/tree/main/manual-install)

**Reply by szaimen** [reply]
Hm...looks like websockets do not work in your case. But cannot repeoduce this. Did you already tried a different browser and/or network?

**Reply by t3cneo** [reply]
Tried Firefox, Safari and Chrome :

I see `WebSocket connection to 'wss://cloud.domain.tld/push/ws' failed: ` on every browser. 

Interestingly onlyoffice is slow to load only with firefox...

What do you mean by have you tried a different network?
Docker network?

**Reply by t3cneo** [reply]
Hey, 

just found out the problem is between the chair and the keyboard...

I updated my backup copy of docker-compose.yml instead of the production one, container `aio-notify-push` was missing from my stack.
